index.html

```diff
@@ -265,7 +265,7 @@ <h4 class="connect">LET'S CONNECT</h4>
       <h2 class="header">Contact me</h2>
       <div class="bodie">
         <div class="text">
-          <p>
+          <p class="para">
             If you have an application you are interested in developing, a
             feature that you need built or a project that needs coding. I’d love
             to help with it
```

Comment:
- [ ] Let's try to follow the design guidelines as shown below 👇 
- [ ] Adjust your contact section to be centered according to the Figma design
- [ ] Please align the submit button to the left of the contact section as shown below 👇 
- [ ] The button interaction state styles should be modeled according to the Figma design
- [ ] Your design is missing the black horizontal bar at the bottom of the contact section. Please reference the design below and implement it 🙏 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/165717316-64e2d4b3-d4f2-45e5-9b27-8ac013c7cfa5.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/165718690-611464ad-2f20-48eb-b0ff-d76886132879.png)